src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [email, setEmail] = useState("");
+    const [birthday, setBirthday] = useState("");
+    const handleSubmit = (event) => { };
```

Comment:
You need to implement the `handleSubmit` method similar to how you did for the `LoginView`.

Reply:
updated code to include additional code